R-package/R/zzz.R

```diff
@@ -16,7 +16,10 @@ NULL
 
 .onLoad <- function(libname, pkgname) {
   # Require methods for older versions of R
-  require(methods)
+  if (! is.element("package:methods", search())) {
```

Comment:
We required R >= 3.3.0 and I think after R 3.3.0 `methods` will always be loaded during the startup.

Reply:
Cool, I hadn't paid attention to what R version was required.. Hadley Wickham has a nice writeup about how to deal with this issue [here](http://r-pkgs.had.co.nz/namespace.html).  He says `Rscript` doesn't automatically load `methods` like an interactive session does, so the following is useful:

```
#' @import methods
NULL
```